src/module/auth/interface/__test__/auth.spec.ts

```diff
@@ -146,4 +138,19 @@ describe('AuthController', () => {
     expect(res.statusCode).toEqual(403);
     expect(res.error).toBeDefined();
   });
+
+  it('Should logout user', async () => {
+    const { header } = await request(app.getHttpServer())
+      .get('/auth/logout')
+      .auth(tokens.adminUserToken, { type: 'bearer' })
+      .set('Cookie', `pizza=${sessionToErase}`)
+      .expect(HttpStatus.OK);
+
+    console.log(header['set-cookie']);
+    expect(header['set-cookie']);
```

Comment:
We should be expecting to compare with something.
Or try to make a refresh call after and check if it fails.